lib/commands/doctor/checks/binary-deps.js

```diff
@@ -1,22 +1,93 @@
-const chalk = require('chalk');
-const {SystemError} = require('../../../errors');
+// @ts-check
```

Comment:
This is the main file that needs to be reviewed, most everything else is from #1207